### PR TITLE
Refactor Paginate IT

### LINKED ISSUE #906: PaginationIT
Jon

### PR DESCRIPTION
**Pull request by jkusa**
Resolves #906

## Description

- Upgrade to rest assured 4.x
- Fix style issues
- Refactor Paginate IT

## Motivation and Context

Refactor IT tests

## How Has This Been Tested?

With Tests

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: elide-integration-tests/src/test/java/com/yahoo/elide/assignedIdLongTests/AssignedIdLongIT.java [hunk 1]
```diff
@@ -5,13 +5,8 @@
...
 package com.yahoo.elide.assignedIdLongTests;
 
-import static com.jayway.restassured.RestAssured.given;
-import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.attr;
-import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.attributes;
-import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.datum;
-import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.id;
-import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.resource;
-import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.type;
+import static com.yahoo.elide.contrib.testhelpers.jsonapi.JsonApiDSL.*;
```
Comment (hellohanchen) [inline elide-integration-tests/src/test/java/com/yahoo/elide/assignedIdLongTests/AssignedIdLongIT.java]: Is`import *` fine here?

File: elide-integration-tests/src/test/java/com/yahoo/elide/tests/UserTypeIT.java [hunk 2]
```diff
@@ -186,25 +181,25 @@ public void testUserTypeMissingUserTypeProperties() throws Exception {
         partialAddress.put("zip", partialZip);
 
         Resource resource = resource(
-                type("person"),
-                id("4"),
-                attributes(
-                        attr("name", "WC"),
-                        attr("address", partialAddress)
-                )
+            type("person"),
```
Comment (hellohanchen) [inline elide-integration-tests/src/test/java/com/yahoo/elide/tests/UserTypeIT.java]: I think we have single tab here but also have double tabs in other places.

File: elide-standalone/src/main/java/com/yahoo/elide/standalone/ElideStandalone.java [hunk 3]
```diff
@@ -121,7 +125,8 @@ public void start(boolean block) throws Exception {
                     "/*",
                     EnumSet.of(DispatcherType.REQUEST));
 
-            context.setAttribute(HealthCheckServlet.HEALTH_CHECK_REGISTRY, ElideResourceConfig.getHealthCheckRegistry());
+            context.setAttribute(HealthCheckServlet.HEALTH_CHECK_REGISTRY,
+                    ElideResourceConfig.getHealthCheckRegistry());
```
Comment (hellohanchen) [inline elide-standalone/src/main/java/com/yahoo/elide/standalone/ElideStandalone.java]: For example here is a double tab.

File: elide-integration-tests/src/test/java/com/yahoo/elide/assignedIdLongTests/AssignedIdLongIT.java (hunk 1 above)
Comment (jkusa) [reply elide-integration-tests/src/test/java/com/yahoo/elide/assignedIdLongTests/AssignedIdLongIT.java]: I think for static testing helpers this is ok.

File: elide-integration-tests/src/test/java/com/yahoo/elide/tests/UserTypeIT.java (hunk 2 above)
Comment (jkusa) [reply elide-integration-tests/src/test/java/com/yahoo/elide/tests/UserTypeIT.java]: Double indent seems to intense. Do you agree?

File: elide-standalone/src/main/java/com/yahoo/elide/standalone/ElideStandalone.java (hunk 3 above)
Comment (jkusa) [reply elide-standalone/src/main/java/com/yahoo/elide/standalone/ElideStandalone.java]: I think that is intellij doing that for line continuations. We should fix all the indents in the project.

File: elide-integration-tests/src/test/java/com/yahoo/elide/tests/UserTypeIT.java (hunk 2 above)
Comment (hellohanchen) [reply elide-integration-tests/src/test/java/com/yahoo/elide/tests/UserTypeIT.java]: I am fine with either way. I just got confused when I saw the inconsistency.

**Pull request merged**